bring-it-up/bring-it-up
# convert fe components to typescript files

## Description
#### Summary:
convert FE components to .txs files

#### Changes:
- typescript conversion for FE components
- add Service class in Service.js file

#### How to test this PR:
- make sure the nav bar is working properly
- make sure the service page is fetching and rendering the service names

## Screenshots:
n/a
